pkgs/applications/misc/ipmiview/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://www.supermicro.com/wftp/utility/IPMIView/Linux/IPMIView_${version}_build.${buildVersion}_bundleJRE_Linux_x64.tar.gz";
-    sha256 = "14cl9jhp2wvv5dgv4akkp75nqjz21f6j9pi6743s3rp298rpj095";
```

Comment:
Could you move this hash change in the previous commit where the version is bumped? I am also OK with squashing the two commits together.

Reply:
done